contrib/gate/README.md

```diff
@@ -1,6 +1,6 @@
 ![PODMAN logo](../../logo/podman-logo-source.svg)
 
 A standard container image for lint-checking and validating changes to the libpod
```

Comment:
I know it's preexisting, but could you change the start of the first sentence to `The "gate" images is a standard container image ...`

Reply:
That sounds better. Thanks.